test/index.js

```diff
@@ -1,126 +1,122 @@
-const test = require('tape')
-const once = require('once')
-const { PassThrough, Transform } = require('readable-stream')
-const endOfStream = require('end-of-stream')
-const pump = require('pump')
-const ObjMultiplex = require('../index.js')
+/* eslint-disable node/no-unpublished-require */
```

Comment:
You should enable this rule. The errors will go away if you add the following to `package.json` (right under the `main` property is good):
```
  "files": ["dist"],
```